### DISCUSSION TITLE
command 'extension.executeAREPLBlock' not found

### DISCUSSION THREAD

**Discussion by antoineba35** [community/community#7483] [Copilot Conversations]
When I do control + enter it gives me the error:
![image](https://user-images.githubusercontent.com/61510528/140823169-819b8d73-1ffb-4357-b83f-61b4a4dc0259.png)

**Comment by Krzysztof-Cieslak** [comment]
It seems you have another extension using the same keyboard shortcut. You can check that with Code->Preferences->Keyboard Shortcuts and search for the reported command ID.
